src/util/time.js

```diff
@@ -175,33 +180,32 @@ function isValidCronPattern (pattern) {
  * @param {String} [attrs.timezone='UTC'] - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
  * @returns {String[]} ISO Dates
  */
-function computeRecurringDates (pattern, { startDate, endDate, timezone = 'UTC' } = {}) {
+function computeRecurringDates (pattern, { startDate, endDate, timezone } = {}) {
```

Comment:
Why are you removing this explicit default (and keeping the comment about default above `[attrs.timezone='UTC']`)

Reply:
Right. I update the JSDoc.

Reply:
You did not tell me why you’re removing the explicit default.

Reply:
Removing the explicit default is a preventive measure against the case where `timezone` is `null`.
This is not to depend on library internal implementation. Currently, passing `null` works because the following condition is falsy:
https://github.com/kelektiv/node-cron/blob/df186541e0ec14a1400c44a053a1c90da98fecbd/lib/cron.js#L27-L33

This measure can probably be removed as `timezone = null` is tested here:
https://github.com/stelace/stelace/blob/2b9ffecc4877e9c73f4fed1148f379af43d01508/test/unit/util/time.spec.js#L165-L168

If so, I'll restore the explicit default value.

Reply:
You should think about what the implicit default value is (due to your change) and how it is different from previous explicit default value.

Explicit is better than implicit.

Reply:
My change was to handle the case where `timezone === null` was passed to the function `computeRecurringDates`. So maybe the best way to handle that case is to restore the explicit default value, add the following code into the function.

```js
// @param {String} [attrs.timezone='UTC'] - https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
// if `null`, `timezone` will also be set to the default value ('UTC')
function computeRecurringDates (pattern, { startDate, endDate, timezone = 'UTC' } = {}) {
  if (isNil(timezone)) timezone = 'UTC'
  ...
}
```

Reply:
Yes, thanks.